lib/core/versions/latest/BatchWriter.ts

```diff
@@ -13,7 +13,7 @@ import Operation from './Operation';
 import OperationType from '../../enums/OperationType';
 import ProtocolParameters from './ProtocolParameters';
 import RecoverOperation from './RecoverOperation';
-import RevokeOperation from './RevokeOperation';
+import DeactivateOperation from './DeactivateOperation';
```

Comment:
Nit: sort for all occurrences... painful I know... but that's the convention.... unless we just say forget about sorting altogether...